### PR TITLE
Add nt210 config to QMK

### PR DESCRIPTION
**Pull request by evyd13**
## Description

Adds the nt210, a custom PCB for the Leopold FC210TP, to QMK.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/evyd13/nt210/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# nt210
+
+![nt120 PCB](https://i.imgur.com/DZAfoOl.jpg)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/evyd13/nt210/readme.md:5]:
```suggestion
A custom PCB for the Leopold FC210TP.
```

Or something.

File: keyboards/evyd13/nt210/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Hardware Supported: nt-120 PCB
+* Hardware Availability: [Open source on GitHub](https://github.com/evyd13/nt-series/tree/main/nt-210)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nt210:default
+
+Flashing example for this keyboard:
+
+    make nt210:default:flash
```
Comment (zvecr) [inline keyboards/evyd13/nt210/readme.md:17]:
```suggestion
    make evyd13/nt210:default

Flashing example for this keyboard:

    make evyd13/nt210:default:flash
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by ikbenkous** [comment]
When using the `default` keymap, `BOOTMAGIC_ENABLE` is not `yes` and thus 'bootmagic lite' is not compiled in. This seems to effectively disable the "press ESC for bootloader during power up" functionality.

Putting `BOOTMAGIC_ENABLE = yes` in the global rules.mk seems to fix it.

File: keyboards/evyd13/nt210/info.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
+{
+    "manufacturer": "Evelien Dekkers",
+    "keyboard_name": "nt210",
+    "maintainer": "evyd13",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic_lite": true,
```
Comment (zvecr) [inline keyboards/evyd13/nt210/info.json:8]:
```suggestion
        "bootmagic": true,
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by evyd13** [comment]
🥺🙏

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**